### DISCUSSION TITLE
Code Formatting

### DISCUSSION THREAD

**Discussion by SebasAran16** [intellij-solidity/intellij-solidity#416] [Q&A]
Hello,

I would like to know how to use the code formatting, or if it either needs for a set up.

I have not been able to format my Solidity code neither with the this plugin nor with Prettier.

Would love to know how to be able to format Solidity Code on PHPStorm.

Thank you very much!

**Comment by SerCeMan** [comment]
Hey, @SebasAran16! Formatting should work out of the box. Could describe the issue in a bit more detail?
